### PR TITLE
edited some errors

### PR DESCRIPTION
**Pull request by ChibiMango**

### CODE REVIEW

**Review by DaanV2 — CHANGES_REQUESTED** [review]
Just wondering about some things, I have no idea if its wrong, or if its done somewhere in the vanilla files

File: source/behavior/entities/format/behaviors/avoid_mob_type.json [hunk 1]
```diff
@@ -75,12 +75,6 @@
       "default": 1.0,
       "description": "Multiplier for walking speed. 1.0 means keep the regular speed, while higher numbers make the walking speed faster."
     },
-    "work_in_rain_tolerance": {
```
[the remark this answers is not in the document]
Comment (ChibiMango) [reply source/behavior/entities/format/behaviors/avoid_mob_type.json:78]: Yes, seems to be copied from the "work_composter" component

File: source/behavior/entities/format/components/block_climber.json [hunk 2]
```diff
@@ -1,31 +1,8 @@
 {
   "$id": "blockception.minecraft.behavior.entities.minecraft.block_climber",
-  "type": "object",
   "title": "Block Climber",
+  "type": "object",
   "additionalProperties": false,
-  "description": "Fires off a specified event when a block in the block list is broken within the sensor range.",
```
[the remark this answers is not in the document]
Comment (DaanV2) [reply source/behavior/entities/format/components/block_climber.json:6]: Cant find anything on block_climber in vanilla too, so marking this as correct 😄

**Review by DaanV2 — APPROVED** [review]

**Review by DaanV2 — APPROVED** [review]
Looks good, 🎉

**Pull request merged**